infra/lib/env-config.ts

```diff
@@ -32,5 +32,6 @@ export interface EnvConfig {
     host: string;
     domain: string;
   };
+  connectWidgetUrl?: string;
```

Comment:
Could do some tweaking on the type definition so one of these is present: `connectWidget` or `connectWidgetUrl`